specification/monitor/resource-manager/microsoft.insights/2017-05-01-preview/examples/listDiagnosticSettings.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters" : {
+    "resourceUri" : "/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6",
+    "api-version" : "2017-05-01-preview"
+  },
+  "responses" : {
+    "200" : {
+      "headers" : {},
+      "body" : {
+        "value": [ 
+          {
+            "id":"/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6/diagnosticSettings/mysetting",
+            "type":"microsoft.logic/workflows",
```

Comment:
The parameter resourceUri starts with a forward slash and the path template: /{resourceUri}/providers/microsoft.insights/diagnosticSettingsCategories/{name} contains a forward slash before the parameter starts. This will cause double forward slashes in the request url. Thus making it incorrect. Please rectify the example.'